### PR TITLE
Add <NAME> arg to help output

### PR DESCRIPTION
**Pull request by heiko-braun**
**What issue type does this pull request address?** (keep at least one, remove the others) 
/kind enhancement

**What does this pull request do? Which issues does it resolve?** (use `resolves #<issue_number>` if possible) 
resolves #
ENG-4070

### CODE REVIEW

File: cmd/vclusterctl/cmd/platform/add/vcluster.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ vcluster platform add vcluster my-vcluster --namespace vcluster-my-vcluster --pr
 	`
 
 	addCmd := &cobra.Command{
-		Use:   "vcluster",
+		Use:   "vcluster VCLUSTER_NAME",
```
Comment (johannesfrey) [inline cmd/vclusterctl/cmd/platform/add/vcluster.go:36]: We could also use this util func that we already use in some places (probably not consistent over the whole code base): https://github.com/loft-sh/vcluster/blob/main/cmd/vclusterctl/cmd/create.go#L35

File: cmd/vclusterctl/cmd/platform/add/vcluster.go [hunk 2]
```diff
@@ -33,7 +33,7 @@ vcluster platform add vcluster my-vcluster --namespace vcluster-my-vcluster --pr
...
 		Short: "Adds an existing vCluster to the vCluster platform",
 		Long:  description,
 		Args:  cobra.ExactArgs(1),
```
Comment (johannesfrey) [inline cmd/vclusterctl/cmd/platform/add/vcluster.go:39]: Also this here: https://github.com/loft-sh/vcluster/blob/main/cmd/vclusterctl/cmd/create.go#L46

File: cmd/vclusterctl/cmd/platform/add/cluster.go [hunk 3]
```diff
@@ -50,7 +50,7 @@ func NewClusterCmd(globalFlags *flags.GlobalFlags) *cobra.Command {
 	}
 
 	c := &cobra.Command{
-		Use:   "cluster",
+		Use:   "cluster CLUSTER_NAME",
```
Comment (johannesfrey) [inline cmd/vclusterctl/cmd/platform/add/cluster.go:53]: If we go the util route we might want to add a `Cluster` entry there as well!?

**Review by FabianKramm — CHANGES_REQUESTED** [review]
We need to be careful here as `Use` is also used to build the command itself

**Comment by heiko-braun** [comment]
Can you elaborate @FabianKramm ?

**Pull request closed without merging**